1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,21 @@
+// Get DOM elements
+const bookContainer = document.querySelector('#reading-list');
+
 function readingList(books) {
   // Write your code here...
+  let result = '';
+  for (let book of books) {
+    let read = "class='not-read'";
+    book.alreadyRead? read = 'class="read"' : read;
+      
+    result += `
+      <li ${read}>
+      <p>${book.title}</p>
+      <p>${book.author}</p>
+      <img src="${book.bookCoverImage}" alt="">
+    </li>`
+  }
+  bookContainer.innerHTML = result;
```

Comment:
Comment: Well done. Let me know how it list what books you read or not. is this what you do in bookContainer?I thought we was not supposed to change the array const books [].

Consider: I just left them in there. https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week3/pull/215/files